filters/auth/authclient.go

```diff
@@ -21,13 +24,16 @@ const (
 	tokenIntrospectionSpan = "tokenintrospection"
 )
 
-const defaultMaxIdleConns = 64
+const (
+	defaultMaxIdleConns = 64
+	tracingTagURL       = "http.url"
```

Comment:
You can reuse the "constant" from https://github.com/opentracing/opentracing-go/blob/master/ext/tags.go#L91

Reply:
I actually mean, not to declare this one and use `ext.HTTPUrl` wherever `tracingTagURL` was referenced

Reply:
@szuecs if you do another iteration on this, tagging token validation calls as `span.kind: client` would be good to have. I think that's also part of `ext/tags` @lmineiro posted.

Reply:
will do

Reply:
I did this in https://github.com/zalando/skipper/pull/1255 which I will merge into this pr. I will push it tomorrow, when I added tests.

Reply:
merged #1255 into this one